ports/rp2/mpconfigport.h

```diff
@@ -238,7 +238,7 @@ extern const struct _mod_network_nic_type_t mod_network_nic_type_wiznet5k;
         extern void mp_handle_pending(bool); \
         mp_handle_pending(true); \
         best_effort_wfe_or_timeout(make_timeout_time_ms(1)); \
-        MICROPY_HW_USBDEV_TASK_HOOK \
+        if (get_core_num() == 0) { MICROPY_HW_USBDEV_TASK_HOOK } \
```

Comment:
There are other calls to `MICROPY_HW_USBDEV_TASK_HOOK` which I think also need to be handled this way.

I'll do a refactor first.

Reply:
See #8863.